frontend/package.json

```diff
@@ -1,13 +1,12 @@
 {
   "name": "frontend",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "@babel/core": "7.18.5",
     "@bcgov/bc-sans": "1.0.1",
     "@react-keycloak/web": "3.4.0",
     "@reduxjs/toolkit": "1.6.0",
-    "@types/react-bootstrap-typeahead": "3.4.7",
```

Comment:
Just curious why react-bootstrap-typeahead and react-responsive were both removed from the package.json file?

Reply:
It seems like before there were 2 separate packages, one for the @types: `"@types/react-bootstrap-typeahead": "3.4.7"` and one for the dependency: `"react-bootstrap-typeahead": "5.1.2"`

and now the dependency has been downgraded to 3.4.7, but now we are missing the @types for react-bootstrap-typeahead? Would this cause an issue?

Reply:
Sorry wrong line number.  I've removed duplicates.